components/layouts/TheMenu/index.vue

```diff
@@ -51,33 +51,33 @@
             :class="{ 'menu__logo_active': isActiveHome(homeLink) }"
           />
         </AppLink>
-        <AppLink
-          v-for="link in mainLevelLinks"
-          :key="link.url"
-          class="menu__link"
-          :class="{ 'menu__link_active': isActive(link) }"
-          v-bind="link"
-          kind="secondary"
-        >
-          {{ link.label }}
-        </AppLink>
-      </nav>
-    </section>
-    <section
-      v-if="isCommunityActive()"
-      class="menu__second-level"
-    >
-      <nav class="menu__navigation-level">
-        <AppLink
-          v-for="link in communitySubLinks"
-          :key="link.url"
-          class="menu__link"
-          :class="{ 'menu__link_active': isActive(link) }"
-          v-bind="link"
-          kind="secondary"
-        >
-          {{ link.label }}
-        </AppLink>
+        <ul class="menu__navigation-level__list">
+          <li v-for="link in mainLevelLinks" :key="link.url" class="menu__navigation-level__list-item">
+            <AppLink
+              v-if="!link.sublinks"
+              class="menu__link"
+              :class="{ 'menu__link_active': isActive(link) }"
+              v-bind="link"
+            >
+              {{ link.label }}
+            </AppLink>
+            <cv-dropdown v-else class="cv-dropdown menu__navigation__dropdown" :class="{ 'menu__link_active': isCommunityActive() }" placeholder="Community">
+              <li
```

Comment:
@y4izus - I tried using the `cv-dropdown-item`, but as pointed out in the code, that component includes an `<a>` element with it... and since we're _slotting in_ our own `app-link` (`<a>`), it then becomes invalid HTML since we can't have nested anchors.

Reply:
<img width="846" alt="Screen Shot 2020-12-08 at 9 54 42 AM" src="https://user-images.githubusercontent.com/6276074/101516091-4c8c0d80-393c-11eb-8c81-100eee9205f3.png">

Reply:
You're right @techtolentino , let's continue with your code :). Thanks for trying it out!!